examples/wallet/app/utils/wasmWrapper.js

```diff
@@ -13,6 +13,25 @@ import { totalParts } from './proportionsHelper';
 const wasmBindings = import('js-chain-libs/js_chain_libs');
 export async function getAccountFromPrivateKey(
   secret: InternalPrivateKey
+): Promise<AccountKeys> {
+  const { PrivateKey } = await wasmBindings;
+  const privateKey: PrivateKey = PrivateKey.from_bech32(secret);
+  return getAccountDataFromPrivateKey(privateKey);
+}
+
+export async function getAccountFromSeed(
+  seed: Uint8Array
+): Promise<AccountKeys> {
+  const { PrivateKey } = await wasmBindings;
+  const privateKey: PrivateKey = PrivateKey.from_normal_bytes(
+    seed.slice(0, 32)
+  );
+
+  return getAccountDataFromPrivateKey(privateKey);
+}
+
+async function getAccountDataFromPrivateKey(
+  privateKey: PrivateKey
```

Comment:
keep in mind this type is not defined in this context.  
For using the type definitions without first instatinating the wasm, we need proper Flow bindings instead of using the classes.  
Until then, we could use `any` type.